setup.py

```diff
@@ -507,6 +507,10 @@ def dummy_import_torch():
 def check_torch_version():
     try:
         import torch
+        if torch.__version__ < '0.4.0':
```

Comment:
Generally safer to use `distutils.version`: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

Reply:
Good idea!